.eslintrc

```diff
@@ -23,14 +23,45 @@
       },
       "rules": {
         "strict": "off",
+        "curly": "off",
         "semi": ["error", "always"],
         "indent": ["error", 2, {"SwitchCase": 1}],
         "quotes": "warn",
         "no-octal": "warn",
         "no-unused-vars": "warn",
         "no-console": "warn",
         "no-undef": "warn",
-        "no-useless-escape": "warn"
+        "no-useless-escape": "warn",
```

Comment:
All of these additions are just turning 'semistandard` errors into warnings for our legacy code so we don't have to fix everything right now.